### PR TITLE
P1 rating

### PR DESCRIPTION
**Pull request by Omkar-Ghate**
For the rating component. Please review the code and design.

### CODE REVIEW

File: pages/components/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+.rating-star-icon{
+    color: rgba(0, 0, 0, 0.26);
```
Comment (zoef-yt) [inline pages/components/rating/rating.css]: use var

File: pages/components/rating/rating.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.rating-star-input:enabled .rating-star-icon{
+    cursor: pointer;
+}
+
+.rating-star-input:enabled + .rating-star-icon:hover{
+    color: #ffd675;
```
Comment (zoef-yt) [inline pages/components/rating/rating.css]: use var

File: pages/components/rating/rating.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    color: #ffb400;
+}
+
+.rating-badge{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: fit-content;
+    font-size: 0.75rem;
+    line-height: 0.75rem;
```
Comment (zoef-yt) [inline pages/components/rating/rating.css]: use px

File: pages/components/rating/rating.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+.rating-star-input{
+    width: 0;
+    margin: 0;
+    height: 0;
+    opacity: 0;
+}
+
+.rating-star-input:checked + .rating-star-icon{
+    color: #ffb400;
```
Comment (zoef-yt) [inline pages/components/rating/rating.css]: use var

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: pages/components/rating/rating.css (hunk 1 above)
Comment (Omkar-Ghate) [reply pages/components/rating/rating.css]: sure, thanks

File: pages/components/rating/rating.css [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.rating-star-input:enabled .rating-star-icon{
+    cursor: pointer;
+}
+
+.rating-star-input:enabled + .rating-star-icon:hover{
+    color: #ffd675;
+}
+
```
Comment (Shubhamkr2610) [inline pages/components/rating/rating.css]: I think you should use color with either color variable name or hex color code to maintain consistency

File: pages/components/rating/rating.css (hunk 1 above)
Comment (Shubhamkr2610) [inline pages/components/rating/rating.css]: you should use either color with variable name or hex color code

File: pages/components/rating/rating.css (hunk 3 above)
Comment (Omkar-Ghate) [reply pages/components/rating/rating.css:31]: I have used px if the value is below 10px or if I want the size to remain constant irrespective of the parent's / root size.

**Pull request merged**